search2/python/search/save/listener.py

```diff
@@ -211,8 +57,29 @@ def main():
 
     with open(args.search_conf, "r", encoding="utf-8") as search_config_file:
         search_conf = YAML(typ="safe").load(search_config_file)
-    listen(queue_conf, search_conf, config_path_base_dir)
 
+    def handler_fn(publisher: Publisher, job_details: dict):
+        input_body = _coerce_inputs(job_details)
+        return go(input_body=input_body, search_conf=search_conf, publisher=publisher)
+
+    def submit_msg_fn(base_msg: dict, job_details: dict):
+        config_path = get_config_file_path(config_path_base_dir, job_details['assembly'])
+
+        with open(config_path, 'r', encoding='utf-8') as f: # pylint: disable=invalid-name
```

Comment:
another instance of this pattern :)  

again, what's with invalid-name here?

Reply:
pylint doesn't like single letter names. single letter names are preferable where they are locally obvious